.github/workflows/ci_windows_x64_msvc.yml

```diff
@@ -22,17 +22,24 @@ concurrency:
 
 jobs:
   windows_x64_msvc:
-    runs-on: windows-2022
-    defaults:
-      run:
-        shell: bash
+    runs-on: arc-runner-set
     env:
-      BUILD_DIR: build-windows
+      BASE_BUILD_DIR_POWERSHELL: C:\mnt\azure\b
     steps:
       - name: "Checking out repository"
         uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
         with:
           submodules: true
+      - name: "Create build dir"
```

Comment:
marking request changes until you're done testing directory creation/cleanup

LMK when this is ready for another review